doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -221,7 +223,15 @@ Bootloader libraries
 Modem libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`lib_location` library:
+
+  * Added:
+
+    * Support for sending Wi-Fi access point list to the cloud by the application.
```

Comment:
```suggestion
    * Support for sending the Wi-Fi access point list to the cloud.
```

Reply:
"by the application" is the key thing here in terms of what's being added. Sending the Wi-Fi list to cloud by this library has been available for months. Now we are adding an event so that Wi-Fi AP list is sent to the application to indicate that send it to cloud or do whatever you want to do with it.